daml-lf/archive/BUILD.bazel

```diff
@@ -226,3 +227,15 @@ pkg_tar(
     package_dir = "daml-lf-archive-protos/protobuf/da",
     visibility = ["//visibility:public"],
 )
+
+da_scala_binary(
```

Comment:
if this is a test, should this be a `da_scala_test`?

Reply:
@dajmaki That's one or you.

Reply:
It should perhaps be called ”decode-tester” or just decoder. It’s an ad-hoc cli tool for benchmarking and profiling the decoding. We could also consider just commenting it out as it’s only used when needed?

Reply:
Renamed it and added a comment.

Reply:
Thanks!